dao/reminders.js

```diff
@@ -97,7 +115,11 @@ module.exports = {
         'SELECT * FROM reminders WHERE family = ? AND id = ?',
         family, id
       ))
-      .then(row => row || Promise.reject(notFoundError(id)));
+      .then(reminder => (
+        reminder
+          ? deserialize(reminder)
+          : Promise.reject(notFoundError(id))
+      ));
```

Comment:
Noted. I got that error too, but I was unsure what were the potential harmful effects